docs/jwst/outlier_detection/main.rst

```diff
@@ -1,58 +1,42 @@
-  
-#. perform a statistical comparison (pixel-by-pixel) between the median blotted
-   data with the original input data to look for pixels with values that are  
-   different from the mean value by more than some specified sigma 
-   based on the noise model
-
-   - the noise model used relies on the error array computed by previous 
-     calibration steps based on the readnoise and calibration errors
-  
-#. flag the DQ array for the input data for any pixel (or affected neighboring
-   pixels) identified as a statistical outlier
-
-The outlier detection step serves as a single interface to apply this general 
-process to any JWST data, with specific variations of this algorithm for each 
-type of data.  Sub-classes of the outlier detection algorithm have been developed
-specifically for:
-
-#. Imaging data
-#. IFU spectroscopic data
-#. TSO data
-#. coronagraphic data
-#. spectroscopic data
-  
-This allows the outlier_detection step to be tuned to the variations in each type 
-of JWST data.
-           
-Reference Files
-===============
-
-The ``outlier_detection`` step uses the PARS-OUTLIERDETECTIONSTEP parameter reference file.
-
-.. include:: ../references_general/pars-outlierdetectionstep_reffile.inc
+or cosmic rays that remain in each of the input images, many times at levels which
+were not detectable by the :ref:`jump <jump_step>` step.
+The ``outlier_detection`` step supports multiple
+algorithms and determines the appropriate algorithm for the type of observation
+being processed.  This step supports:
+
+* **Image modes**: 'FGS_IMAGE', 'MIR_IMAGE', 'NRC_IMAGE', 'NIS_IMAGE'
+  - See :ref:`outlier-detection-imaging` for algorithm details
+* **Slit-like Spectroscopic modes**: 'MIR_LRS-FIXEDSLIT', 'NRS_FIXEDSLIT', 'NRS_MSASPEC'
+   - See :ref:`outlier-detection-spec` for algorithm details
+* **Time-Series-Observation (TSO) modes**: 'MIR_LRS-SLITLESS', 'NRC_TSGRISM', 'NIS_SOSS', 'NRS_BRIGHTOBJ', 'NRC_TSIMAGE'
+   - See :ref:`outlier-detection-tso` for algorithm details
+* **IFU Spectroscopic modes**: 'MIR_MRS', 'NRS_IFU'
+   - See :ref:`outlier-detection-ifu` for algorithm details
+* **Coronagraphic Image modes**: 'MIR_LYOT', 'MIR_4QPM', 'NRC_CORON'
+   - See :ref:`outlier-detection-coron` for algorithm details
+
+This step uses the following logic to apply the appropriate algorithm to the
+input data:
+
+#. Interpret inputs (Association, ModelContainer, ModelLibrary, or CubeModel)
+   to identify all input observations to be processed
+
+#. Read in parameters set by user. See :ref:`outlier_detection_step_args` for the full list
+   of parameters.
+
+#. Select outlier detection algorithm based on exposure type in input model ``meta.exposure.type``.
+
+#. Instantiate and run outlier detection class determined for the exposure type
+   using parameter values interpreted from inputs.
+
+#. Output models will have DQ arrays updated with flags for identified outliers.
+
+.. automodapi:: jwst.outlier_detection.outlier_detection_step
```

Comment:
I don't think this needs to be here - the API is already documented at the end of the index page.